sycl/test-e2e/ESIMD/unified_memory_api/atomic_update_usm_pvc_cmpxchg.cpp

```diff
@@ -0,0 +1,17 @@
+//==------- atomic_update_usm_pvc_cmpxchg.cpp- DPC++ ESIMD on-device test --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: gpu-intel-pvc
+
+// TODO: understand why the test is hanging.
```

Comment:
Is it because of the 'double' type?

Reply:
There was a mess with order with operands in my initial implementation.